portfolio/src/main/webapp/script.js

```diff
@@ -63,18 +63,35 @@ function getCommentSectionFromServer() {
 function createIndividualComment(comment) {
   const liElement = document.createElement('li');
   // This container is used for showing the username.
-  const usernameElement = document.createElement('p');
+  const usernameElement = document.createElement('div');
+  usernameElement.setAttribute('class', 'username-container');
   // This container is used for showing the text of the comment.
-  const commentElement = document.createElement('p');
+  const commentElement = document.createElement('div');
+  commentElement.setAttribute('class', 'comment-text');
+  // Made an image represent the delete button.
+  const deleteButtonElement = document.createElement('img');
+  deleteButtonElement.setAttribute('class', 'delete-button');
+  deleteButtonElement.src = 'images/trash.png';
+  deleteButtonElement.addEventListener('click', () => {
+    deleteComment(comment, liElement);
+  });
 
   usernameElement.innerText = comment.username + ' wrote:';
   commentElement.innerText = comment.text;
-  commentElement.style.color = 'rgb(102, 0, 102)';
-  commentElement.style.fontStyle = 'italic';
-  commentElement.style.fontWeight = 'initial';
-  commentElement.style.height = '100%';
+  liElement.appendChild(deleteButtonElement);
   liElement.appendChild(usernameElement);
   liElement.appendChild(commentElement);
-  liElement.style.border = 'dotted pink';
   return liElement;
 }
+
+function deleteComment(comment, element) {
+  let resultStatus;
+  const params = new URLSearchParams();
+  params.append('id', comment.id);
+  fetch('/delete-comment', {method: 'POST', body: params}).then(result => {
+    //Check if the comment was deleted succesfully.
+    if (result.status === 200) {
```

Comment:
Potentially add a TODO to handle errors correctly.

You don't need to fix TODOs immediately (and adding a TODO does not even bind you from ever fixing it) but it signals to a future reader (who may be wondering why is there no error handling here), that yes, the error handling was not implemented (yet).

Reply:
But in this case, what should be done to handle the possible error? The comment should not be deleted in that case

Reply:
I meant, you probably want to inform the user somehow that there was an error.
Yes, the comments should not be deleted :)